### ISSUE TITLE
0.6.1 update

### ISSUE THREAD

**Issue by fenwick67** [jenil/bulmaswatch#24]
Bulma recently moved to 0.6 and there are a few updates.

The biggest changes that will effect Bulmaswatch is the new blue `link` color, and the navbar markup changed.

https://bulma.io/2017/10/09/roses-are-red-links-are-blue/

**Comment by jenil** [comment]
Yes, I am aware. Its a lot of work and I am gradually updating each theme. You can check the progress [here](https://github.com/jenil/bulmaswatch/tree/0.6.0).

**Comment by ghost** [comment]
@jenil Since you're in the process of updating themes, here are a few things I noticed that need fixing.

**First** the navbar, when it changes to responsive mode the navbar links look like this (see image below).

![4b7ccaf6-df38-453c-b62e-de490e4fee71](https://user-images.githubusercontent.com/8528269/33907441-25aca248-dfc0-11e7-85e3-aab47060ffaa.png)

Fixed by doing this:

![e7857621-7b23-4a8b-b846-4497d583c2fb](https://user-images.githubusercontent.com/8528269/33907872-95447b98-dfc1-11e7-8099-9eaceb5ae025.png)

**Second** is the form controls with icons. When I used Bulmaswatch in combination with [Buefy](https://github.com/rafaelpimpa/buefy), Bulmaswatch's CSS disabled the show/hide toggler on Buefy's password fields.

Apparently it was because Bulmaswatch was disabling `pointer-events` with a high-specificity selector `.control.has-icons-right .icon`. You should just make it `.control .icon`.

![563ebb6c-f1b5-4a11-b2de-b12b4d31411d](https://user-images.githubusercontent.com/8528269/33908555-c68d8c2e-dfc3-11e7-948e-f062b7a00314.png)

**Comment by jenil** [comment]
@raniesantos noted. I am aware of the first point. Thanks for pointing out the second one, I'll fix that.

**Comment by jenil** [comment]
hey @raniesantos I was debugging the form controls with icons and realized that the selector you mentioned is coming from Bulma and has nothing to with Bulmaswatch. In fact, I believe the `is-clickable` selector is something that is Buefy specific and does not come with Bulma. Can you confirm this?

**Comment by ghost** [comment]
@jenil Ok, so I have this in my scss file:

```scss
// Buefy
@import "node_modules/buefy/lib/buefy.min";

// Bulmaswatch
@import "node_modules/bulmaswatch/cyborg/bulmaswatch.min";
```

As you can see, Bulma isn't in here. It just works that way, I assume that the Bulma base styles are just included in Buefy by default.

I checked it all again and **when I comment out the Bulmaswatch import**, the eyeball icon on the password field becomes functional again, it really won't work with Bulmaswatch included.

About `is-clickable`, yes its definitely a Buefy selector.

___

**EDIT:**

I solved it by changing my imports. So the problem was that I was importing the compiled `.css` files (which I initially did because it was easier to pick out what files I needed). I had to figure out which `.scss`/`.sass` files to import and in what order.

I ended up with this:

```scss
// Bulma + Bulmaswatch
@import "node_modules/bulmaswatch/cyborg/variables";
@import "node_modules/bulma/bulma";
@import "node_modules/bulmaswatch/cyborg/overrides";

// Buefy
@import "node_modules/buefy/src/scss/buefy";
```

**Comment by jenil** [comment]
I am sorry this took so long. I just published [v0.6.0](https://github.com/jenil/bulmaswatch/releases/tag/v0.6.0).

Will soon update to 0.6.1 and 0.6.2.

**Comment by raniesantos** [comment]
Can't complain, you do thisnfor free anyway.

**Comment by ghost** [comment]
@jenil 
This line you added at the top of `_variables.scss` for the **Cyborg** theme broke it.

```css
@import 'sass/utilities/all';
```


I checked the other variable files in the other themes and this line doesn't exist. If I remove it manually I get missing variable errors.

**Comment by jenil** [comment]
@raniesantos thanks for reporting, that was a new technique I was trying to import the default Bulma variables, but I guess it breaks the build process for a few people. I have reverted it and should be fine now.

**Comment by jenil** [comment]
Published [0.6.1](https://github.com/jenil/bulmaswatch/releases/tag/v0.6.1) 🎉 

I'd appreciate if you guys test it thoroughly and LMK if you have issues
